src/System.Management.Automation/engine/NativeCommandProcessor.cs

```diff
@@ -698,8 +708,7 @@ internal override void Complete()
                     _nativeProcess.WaitForExit();
 
                     // Capture screen output if we are transcribing
-                    if (this.Command.Context.EngineHostInterface.UI.IsTranscribing &&
-                        _scrapeHostOutput)
+                    if (_isTranscribing && (true == s_supportScreenScrape))
```

Comment:
I think there is a change of behavior here. Assuming `_isTranscribing == $true` and screen scraping is supported,
- before the change, `_scrapeHostOutput` is true only if `_runStandAlone` is true, so this block will run only the native command is running standalone.
- after the change, even if it's not running standalone (i.e. standard input/error redirected), this block of code still will run when transcribing, and in such case, `_startPosition` is not even initialized.

Reply:
Will fix